apps/authentication/admin.py

```diff
@@ -0,0 +1,15 @@
+"""
+
+admin file
+
+This file is subject to the terms and conditions defined in file 'LICENSE',
+which is part of this source code package.
+"""
+
+from sqladmin import ModelView
+
+from apps.authentication.models.api_key import APIKey
+
+
+class ApiKeyAdmin(ModelView, model=APIKey):
```

Comment:
Add docstrings, please make sure you run the ./scripts/lint.sh linter bash script before sending changes.